codec-http2/src/test/java/io/netty/handler/codec/http2/Http2ClientUpgradeCodecTest.java

```diff
@@ -43,7 +51,8 @@ public void testUpgradeToHttp2FrameCodec() throws Exception {
 
     @Test
     public void testUpgradeToHttp2MultiplexCodec() throws Exception {
-        testUpgrade(Http2MultiplexCodecBuilder.forClient(new HttpInboundHandler()).build());
+        testUpgrade(Http2MultiplexCodecBuilder.forClient(new HttpInboundHandler())
+            .withUpgradeStreamHandler(new UpgradeStreamHandler()).build());
```

Comment:
You could also just use `ChannelInboundHandlerAdapter` and so not need to declare an extra class .